docs/nuclei/get-started.md

```diff
@@ -495,6 +514,16 @@ Including request/response in the markdown report is optional, and included when
 nuclei -l urls.txt -t cves/ -irr -markdown-export reports
 ```
 
+**<ins>Sarif Export</ins>**
+
+Nuclei supports Sarif export of valid findings with `-se, -sarif-export` flag. This flag takes a file as input to store Sarif formatted report.
+
+The request-responses are contained as markdown formatted data in Sarif response section.
```

Comment:
How about something like:

The request-response pairs are formatted using markdown syntax and ar e stored within the SARIF response section.